### PR TITLE
Adding Go labs for EKS

### LINKED ISSUE #28: Implement AWS Lab 4 - Kubernetes -  in Go

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #28

### CODE REVIEW

File: labs/aws/go/lab-04/README.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+> This lab assumes you have a project set up. If you don't yet, please [complete this lab first](../01-iac/01-creating-a-new-project.md).
+
+## Step 1 &mdash; Creating a Kubernetes Cluster
+
+> If you do not have an EKS cluster, you can create one by using the code [here](./code/step1.go).
+
+Point the `KUBECONFIG` environment variable at your cluster configuration file:
+
+```bash
+export KUBECONFIG=~/iac-workshop/kubeconfig
```
Comment (metral) [inline labs/aws/go/lab-04/README.md:20]:
```suggestion
pulumi stack output kubeconfig > ~/iac-workshop/kubeconfig.json
export KUBECONFIG=~/iac-workshop/kubeconfig.json
```

File: labs/aws/go/lab-04/README.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+CoreDNS is running at https://abcxyz123.gr7.eu-central-1.eks.amazonaws.com/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+```
+
+## Step 2 &mdash; Add a func that generates the Kubeconfig
+
+In your project, add the following function:
+
+```go
+//Create the KubeConfig Structure as per https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html
+func generateKubeconfig(clusterEndpoint pulumi.StringOutput, certData pulumi.StringOutput, clusterName pulumi.StringOutput) pulumi.StringOutput {
```
Comment (metral) [inline labs/aws/go/lab-04/README.md]: Step 1 refers to a kubeconfig from a new cluster. Should we consider moving this from step 2 into step1?

File: labs/aws/go/lab-04/code/step8.go [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+					ToPort:     pulumi.Int(0),
+					CidrBlocks: pulumi.StringArray{pulumi.String("0.0.0.0/0")},
+				},
+			},
+			Ingress: ec2.SecurityGroupIngressArray{
+				ec2.SecurityGroupIngressArgs{
+					Protocol:   pulumi.String("tcp"),
+					FromPort:   pulumi.Int(80),
+					ToPort:     pulumi.Int(80),
+					CidrBlocks: pulumi.StringArray{pulumi.String("0.0.0.0/0")},
```
Comment (metral) [inline labs/aws/go/lab-04/code/step8.go]: Don't believe this is needed for any of the steps

Comment (stack72) [reply labs/aws/go/lab-04/code/step8.go]: This is to access the app as port 80 is exposed by the service

**Pull request merged**